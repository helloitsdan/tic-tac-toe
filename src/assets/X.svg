<svg title="X" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 100 100" xml:space="preserve">
  <style>
    .st0{fill:none;stroke-width:8;stroke-linecap:round;stroke-miterlimit:10}
  </style>
	<line class="st0 line" x1="11.22" y1="88.1" x2="88.3" y2="11.02"/>
  <line class="st0 line" x1="88.3" y1="88.1" x2="11.22" y2="11.02"/>
</svg>
