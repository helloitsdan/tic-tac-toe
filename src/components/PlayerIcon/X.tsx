import React, { FunctionComponent } from 'react'
import { ReactComponent as XIcon } from '../../assets/X.svg'

const X: FunctionComponent = () => (
  <XIcon title="X" className="c-player-piece" />
)

export default X
