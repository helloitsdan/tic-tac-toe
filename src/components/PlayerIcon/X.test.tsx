import React from 'react'
import { render } from '@testing-library/react'

import X from './X'

describe('X', () => {
  it('renders', () => {
    const { asFragment } = render(<X />)
    expect(asFragment()).toMatchSnapshot()
  })
})
