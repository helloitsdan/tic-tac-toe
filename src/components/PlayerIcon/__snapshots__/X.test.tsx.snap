// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`X renders 1`] = `
<DocumentFragment>
  <svg
    class="c-player-piece"
    title="X"
  >
    X.svg
  </svg>
</DocumentFragment>
`;
